browser/scripts/application.js

```diff
@@ -1091,14 +1091,21 @@ Application.prototype.calculateCanvasArea = function() {
 	var width, height
 	var isFullscreen = !!(document.mozFullScreenElement || document.webkitFullscreenElement)
 
+	var ww, lnw, mpw, mphw, lphw, rpw, rphw;
+
 	if (!isFullscreen) {
-		width = $(window).width() -
-			$('#left-nav').outerWidth(true) - 
-			$('#mid-pane').outerWidth(true) - 
-			$('.mid-pane-handle').outerWidth(true) - 
-			$('.left-pane-handle').outerWidth(true) -
-			$('#right-pane').outerWidth(true) - 
-			$('.right-pane-handle').outerWidth(true)
+		ww = $(window).width();
+		lnw = $('#left-nav').outerWidth(true);
+		mpw = $('#mid-pane').outerWidth(true);
+		mphw = $('.mid-pane-handle').outerWidth(true);
+		lphw = $('.left-pane-handle').outerWidth(true);
+		rpw = $('#right-pane').outerWidth(true);
+		rphw = $('.right-pane-handle').outerWidth(true);
+		width = ww - lnw - mpw - mphw - lphw - rpw - rphw;
```

Comment:
I'm not a big fan of abbreviated variable names.. https://github.com/airbnb/javascript/tree/master/es5#naming-conventions